packages/create-solid/src/index.ts

```diff
@@ -3,5 +3,24 @@ import { handleNew } from "@solid-cli/commands/new";
 import color from "picocolors";
 import { intro } from "@clack/prompts";
 import { version } from "../package.json";
+import { program } from "commander";
+import { t } from "@solid-cli/utils";
+
 intro(`\n${color.bgCyan(color.black(` Create-Solid v${version}`))}`);
-handleNew();
+
+program
+	.allowExcessArguments(false)
+	.configureHelp({
+		commandUsage: () => `create-solid [${t.OPTIONS}]`,
+	})
+	.helpOption(undefined, t.SHOWS_THIS_HELP_MESSAGE)
+	.option(`-p, --project-name <${t.VALUE}>`, t.NAME_OF_YOUR_PROJECT)
```

Comment:
Many CLIs like this one often allow the project name/path as a positional argument instead of an optional one like I did here.

If that's preferred please let me know 🙂